woocommerce/woocommerce
# New marketplace addons header

### All Submissions:






### Changes proposed in this Pull Request:

Update the Marketplace/Addons header to match new designs.

Just noting that stylint is failing due to an old file and not changes I worked on.

Closes 10864-gh-Automattic/woocommerce.com

### How to test the changes in this Pull Request:

1. Checkout branch
2. Run CSS build `node_modules/grunt/bin/grunt css`
3. Visit `WooCommerce -> Extensions` menu in WP Admin
4. Check the new header matches designs.

![](https://d.pr/i/4xOnvN+)

### Other information:
